apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -193,7 +193,31 @@ async function getFeaturedConsultations(consultation, playlistItems) {
   return featured;
 }
 
-async function processPageConsultation(page) {
+async function processPageConsultationDocument(page, api, context) {
+  const { query } = context;
+
+  const { title, ...rest } = query;
+
+  return {
+    ...page,
+    blocks: [
+      {
+        slug: "embedded-document-viewer",
+        options: rest,
+        title,
+      },
+    ],
+  };
+}
+
+async function processPageConsultation(page, api, context) {
+  const { params } = context;
+
+  // Check if we are on a document page: /opportunities/consultation/documents
+  if (params.slugs.length > 2 && params.slugs[2] === "documents") {
```

Comment:
@kelvinkipruto can we guarantee that params.slug will always have a length of 3?